src/test/java/im/tox/tox4j/AliceBobTestBase.java

```diff
@@ -271,7 +271,7 @@ public void runAliceBobTest_TCP6() throws Exception {
             @NotNull
             @Override
             public ToxCore make() throws ToxException {
```

Comment:
Can you move the `@NotNull` closer to the type? I don't feel strongly about this being a separate change. If you don't want to pollute this PR with formatting changes, that's OK. We'll need a larger cleanup PR later on, anyway.